apps/src/templates/sectionAssessments/MultipleChoiceOverviewTable.jsx

```diff
@@ -57,16 +57,19 @@ const answerDataPropType = PropTypes.shape({
   isCorrectAnswer: PropTypes.bool,
 });
 
-const questionDataPropType = PropTypes.shape({
+const multipleChoiceDataPropType = PropTypes.shape({
   id: PropTypes.number.isRequired,
   question: PropTypes.string.isRequired,
   answers: PropTypes.arrayOf(answerDataPropType),
   notAnswered: PropTypes.number.isRequired,
 });
 
+// A single table that shows students' responses to each multiple choice question.
+// The table displays the percent of students that select an answer choice and
+// percent of students that did not answer the question.
```

Comment:
nit: multi-line comments in js should look like this:
```
/**
 * A single table that shows students' responses to each multiple choice question.
 * The table displays the percent of students that select an answer choice and
 * percent of students that did not answer the question.
 */
```

Reply:
Fixed.